bert/README.md

```diff
@@ -0,0 +1,233 @@
+# Welcome!
+
+This benchmark covers pre-training and fine-tuning a BERT model from scratch. With this starter code, you'll be able to do Masked Language Modeling (MLM) pre-training on the C4 dataset and classification fine-tuning on the 8 different GLUE benchmark tasks. We also provide the source code and recipe behind our Mosaic BERT model, which you can train yourself using this repo.
```

Comment:
"8 different GLUE benchmark tasks", should probably note that we exclude WNLI

Reply:
Fixing this here by just removing detail (since it's an intro paragraph). The WNLI exclusion detail will be mentioned below.